README.md

```diff
@@ -6,7 +6,7 @@
 
 exccstakepool is a web application which coordinates generating 1-of-2 multisig
 addresses on a pool of [exccwallet](https://github.com/EXCCoin/exccwallet) servers
-so users can purchase [proof-of-stake tickets](https://docs.excc.co/mining/proof-of-stake/)
```

Comment:
Here i would also change this one to https://exchangecoin.zendesk.com/hc/en-us/articles/360004934231, because all "support" is redirected there. Besides that, target link doesn't exist

Reply:
Yes, there are also two more links from `views/tickets.html` file. I am waiting for substitute proposition. 
Changes are coming! ;)